packages/client/src/components/UploadZip/UploadZip.tsx

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import { MessageBarType } from '@fluentui/react'
+import React, { memo, useEffect, useCallback, useState } from 'react'
+import { MessageContainer } from '~components/MessageContainer'
+import { Container } from '~styles'
+
+interface UploadZipProps {
+	id: string
+	onUpload: (files: File[]) => void
+}
+
+export const UploadZip: React.FC<UploadZipProps> = memo(function UploadZip({
```

Comment:
I think this is ok for now, but as we extract to a separate component I'd like to dig into dropzone a bit more and see if we can avoid wrapping a raw HTML element like this. Using a Fluent button would be cleaner, especially if it means we can avoid calling the input.click function later.